docs/about/history/index.html

```diff
@@ -14,10 +14,13 @@
 <div>
 	<h2>History of the Web of Things at the W3C</h2>
 
-	<!-- <figure style="float:right;">
-		<img src="/WoT/images/ws2016-overview1.png" width="100%" alt="initial ideas about wot">
-		<figcaption style="text-align: center;">Initial overview figure of the Web of Things prepared for the first W3C WoT Workshop</figcaption> 
-	</figure> -->
+	<br>
+	<figure style="float:middle;">
+		<img src="/WoT/images/timeline.drawio.svg" width="100%" alt="Timeline of the development of WoT at the W3C">
```

Comment:
In other places we use  a different style/form for including  images.

see for example https://github.com/w3c/wot-marketing/blob/4472f4fe9836afe81d30c4c4832a43ce3fe6acdc/docs/documentation/index.html#L38

I wonder whether we should do the same here. I think there was a reason for doing so.

Reply:
@danielpeintner you are right, I have changed it now